mysql-test/main/alter_table.result

```diff
@@ -2448,11 +2448,12 @@ Table	Create Table
 t1	CREATE TABLE `t1` (
   `a` int(11) NOT NULL,
   `new_b` int(11) NOT NULL,
-  `c` int(11) DEFAULT (`a` + `new_b`) CHECK (`a` + `new_b` > 0),
+  `c` int(11) DEFAULT (`a` + `new_b`),
   `d` int(11) GENERATED ALWAYS AS (`a` + `new_b`) VIRTUAL,
   `b` char(1) DEFAULT NULL,
   PRIMARY KEY (`a`),
   KEY `b` (`new_b`),
+  CONSTRAINT `CONSTRAINT_1` CHECK (`a` + `new_b` > 0),
```

Comment:
I don't understand. Why did you change how constraints are printed?

Reply:
Because of `mysqldump`  that uses `show create table` and added constraint for `json_valid`  that for the test `mysqldump.test` returned duplicate error constraint (duplicating when dumping `mysql.global_priv` table).

Reply:
Have updated with the new commit as well PR description with questions and problems for the patch.

Reply:
The patch should be functional - no problems for the patch :).